lib/importproject.cpp

```diff
@@ -254,7 +256,7 @@ void ImportProject::importSln(std::istream &istr, const std::string &path)
         if (pos == std::string::npos)
             continue;
         const std::string vcxproj(line.substr(pos1+1, pos-pos1+7));
-        importVcxproj(path + Path::fromNativeSeparators(vcxproj), variables, emptyString);
```

Comment:
an alternative is to check if vcxproj is an absolute path. something like

    std::string vcxproj(line.substr(pos1+1, pos-pos1+7));
    if (!Path::isAbsolute(vcxproj))
        vcxproj = path + vcxproj;
    importVcxproj(Path::fromNativeSeparators(vcxproj), variables, emptyString);

I don't know if that is better or worse what is your opinion?